lib/wallaroo/core/initialization/local_topology.pony

```diff
@@ -1626,6 +1684,18 @@ actor LocalTopologyInitializer is LayoutInitializer
   be source_ids_query(conn: TCPConnection) =>
     _router_registry.source_ids_query(conn)
 
+  be state_entity_query(conn: TCPConnection) =>
+    _router_registry.state_entity_query(conn)
+
+  be stateless_partition_query(conn: TCPConnection) =>
+    _router_registry.stateless_partition_query(conn)
+
+  be state_entity_count_query(conn: TCPConnection) =>
+    _router_registry.state_entity_count_query(conn)
+
+  be stateless_partition_count_query(conn: TCPConnection) =>
+    _router_registry.stateless_partition_count_query(conn)
```

Comment:
call `.stateless_partitions_count_query` (and rename on `RouterRegistery`)

Reply:
see #2268